scripts/test-upgrade.sh

```diff
@@ -129,4 +129,7 @@ bash scripts/deploy-rpc-playbooks.sh
 #              For a detailed description, please see the README in
 #              the rpc_post_upgrade role directory.
 cd ${RPCD_DIR}/playbooks
+openstack-ansible -e 'logging_upgrade=true' logstash.yml
```

Comment:
I didn't relaize the need for upgrade variables when we broke out the RPCO playbooks into it's own script,[ ``deploy-rpc-playbook.sh``](https://github.com/rcbops/rpc-openstack/blob/master/scripts/test-upgrade.sh#L158)
This would be the second time ``test-upgrade.sh`` would run ``logstash.yml`` and ``elasticsearch.yml``. Once in ``deploy-rpc-playbooks.sh``, and again here.
The tricky part is that we can't just change the way ``logstash.yml`` and ``elasticsearch.yml`` are invoked in ``deploy-rpc-playbooks.sh``, because that script is also used by ``deploy.sh``.  The way I see it, we have these options:

* Write logic into ``deploy-rpc-playbooks.yml`` to run ``openstack-ansible -e 'logging_upgrade=true' logstash.yml`` when it's being invoked by ``test-upgrade.sh``, and ``openstack-ansible logstash.yml`` when it's being invoked by ``deploy.sh``. I don't really like this approach.
* Add code to write these upgrade variables into a temporary ``user_variable_*.yml`` file in here, and keep the invocation of ``deploy-rpc-playbook.sh`` the same. Then delete the temporary ``user_variables_*.yml`` file after the upgrade(maybe as part of the post-upgrade role). This is what support did in their k->l maintenance plan.
3. Just let the playbooks run a second time.

If you're okay with it, I can submit a commit to address it.